Small Floodfill Change (#2064)
cpinitiative/usaco-guide: content/3_Silver/Flood_Fill.mdx

visited array, dimensions, and the current size variable globally.

This means that we want to recursively call the search function for the squares

> above, below, and to the left and right of our current square. Due to it's

Reviewer: ```suggestion
above, below, and to the left and right of our current square. Due to its
```

> recursive nature, you can think of floodfill as a modified version of

Reviewer: Second person isn't used at all in the explanation, so for consistency:
```suggestion
recursive nature, floodfill can be thought of as a modified version of
```

</Warning>

> A non-recursive implementation of flood fill adds the edgesto a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:

Reviewer 2: > edgesto

typo?

> A non-recursive implementation of flood fill adds the edges to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:

Reviewer: I'm pretty sure you add nodes, not edges to the stack/queue in floodfill. This is also the case in the code below.
```suggestion
A non-recursive implementation of flood fill adds adjacent nodes to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```
